public/.well-known/assetlinks.json

```diff
@@ -3,19 +3,9 @@
     "relation": ["delegate_permission/common.handle_all_urls"],
     "target": {
       "namespace": "android_app",
-      "package_name": "exchange.dydx.trading.debug",
+      "package_name": "trade.opsdao.dydxchain",
```

Comment:
This is about to go to dydxprotocol repo, right? Won't we need the original IDs?